src/commands/merge-with-parent.yml

```diff
@@ -23,8 +35,8 @@ steps:
                 echo "Can't merge branch into itself! Skipping."
                 exit 0
               fi
-              git checkout <<parameters.parent>>
-              git merge $CHILD_BRANCH
+              git checkout -f -B parent <<parameters.parent>>
+              git merge --no-ff --no-commit $CHILD_BRANCH
```

Comment:
@failedguidedog - I have some concerns here, especially as a minor release. These flags will alter the resulting state of the repository that other users may be expecting that is not backward compatible. For example, users may be expecting a commit to be completed after this command completes. Others may also be expecting the fast-forward in their workflows, as opposed to a 'no fast-forward'. You also will end up on `parent` branch as opposed to the child branch.

Because of these concerns, would it make sense to create a separate command, e.g. `no-commit-merge-with-parent` or something that won't potentially break other's workflow.

We could also release this as a major change if we think these changes are best to replace this command

Reply:
Would prefer parameters > a new command in this case, to avoid redundancy.

Reply:
@dsayling Thank you for the feedback. You make some good points. As @gmemstr suggested, I've made all the behavior-changing flags optional with parameters.